packages/ListBox/stories/examples/Keyboard/index.js

```diff
@@ -0,0 +1,337 @@
+              <ListBox.RawItem
+                onClick={() => {
+                  console.log("RawItem clicked");
+                }}
+              >
+                Odysseus
+              </ListBox.RawItem>
+              <ListBox.Divider>Anti-heroes</ListBox.Divider>
+              {characters.antiHeroes}
+              <ListBox.Divider>Heroes</ListBox.Divider>
+              {characters.heroes}
+              <ListBox.Divider>Villians</ListBox.Divider>
+              {characters.villians}
+            </ListBox>
+          </Card>
+          <h3>Multi</h3>
+
+          <Card>
+            <ListBox isMulti>
+              <ListBox.Filter />
+              <ListBox.Divider>Raw-Items</ListBox.Divider>
+              <ListBox.RawItem
+                onClick={() => {
+                  console.log("RawItem clicked");
+                }}
+              >
+                Achilles
+              </ListBox.RawItem>
+              <ListBox.RawItem
+                onClick={() => {
+                  console.log("RawItem clicked");
+                }}
+              >
+                Odysseus
+              </ListBox.RawItem>
+              <ListBox.Divider>Anti-heroes</ListBox.Divider>
+              {characters.antiHeroes}
+              <ListBox.Divider>Heroes</ListBox.Divider>
+              {characters.heroes}
+              <ListBox.Divider>Villians</ListBox.Divider>
+              {characters.villians}
+            </ListBox>
+          </Card>
+        </Block>
+        <Block>
+          <CheckListItem>
+            The user is allowed to navigate over disabled elements but can not selected them or interact with them. This
+            might required further discussion of what we really want to do.
+          </CheckListItem>
+        </Block>
+      </Blocks>
+    </>
+  );
+}
+
+export function WithInline() {
+  return (
+    <>
+      <h2>With Inline stories</h2>
+      <Blocks>
+        <Block>
+          <h3>Single</h3>
+
+          <Card>
+            <ListBox isInline>
+              <ListBox.Divider>Anti-heroes</ListBox.Divider>
+              {characters.antiHeroes}
+              <ListBox.Divider>Heroes</ListBox.Divider>
+              {characters.heroes}
+              <ListBox.Divider>Villians</ListBox.Divider>
+              {characters.villians}
+            </ListBox>
+          </Card>
+
+          <h3>Multi</h3>
+          <Card>
+            <ListBox isMulti isInline>
+              <ListBox.Divider>Anti-heroes</ListBox.Divider>
+              {characters.antiHeroes}
+              <ListBox.Divider>Heroes</ListBox.Divider>
+              {characters.heroes}
+              <ListBox.Divider>Villians</ListBox.Divider>
+              {characters.villians}
+            </ListBox>
+          </Card>
+        </Block>
+        <Block>
+          <CheckListItem>
+            When the ListBox is on inline state, the keyboard navigation should behave exactly to any of the previous
+            examples
```

Comment:
An inconsistency I noticed: The inline single version doesn't put keyboard focus on the selected item.  It starts on the container, the arrowing down _sets_ the selection to the first element.

Actually, this got me reconsidering the interaction of automatically selecting the focussed item for the single select.  It means for screen reader users there's no way to simply "browse" the options without losing the pre-existing selection.  But changing this behaviour is probably out of scope for this PR, so I'm also okay with leaving the inline version as-is for now.

cc @oscarkwan @sunminsarahlee